addons/website_form/static/src/snippets/s_website_form/000.js

```diff
@@ -277,5 +286,16 @@ odoo.define('website_form.s_website_form', function (require) {
             // willStart for public widgets...
             this.__started.then(() => $result.replaceWith(qweb.render(`website_form.status_${status}`)));
         },
+
+        //----------------------------------------------------------------------
+        // Private
+        //----------------------------------------------------------------------
+
+        /**
+         * @private
+         */
+        _addRecaptchaLegal: function () {
+            this.$target.append(qweb.render(`website_form.recaptcha_legal`));
+        }
```

Comment:
This should be the base implementation of website to me: receive a $target where to put the legal stuff and add it inside. But the template is then in website. Won't it be ok?

Reply:
Yes, the message will not be custom then but I think it does not matter.